tracer/src/Datadog.Trace.Coverage.collector/AssemblyProcessor.cs

```diff
@@ -477,6 +531,29 @@ private static Instruction CloneInstruction(Instruction instruction)
             };
         }
 
+        private string GetDatadogTracer(TracerTarget tracerTarget)
+        {
+            // Get the Datadog.Trace path
+            string targetFolder = "net461";
+            switch (tracerTarget)
+            {
+                case TracerTarget.Net461:
+                    targetFolder = "net461";
+                    break;
+                case TracerTarget.Netstandard20:
+                    targetFolder = "netstandard2.0";
+                    break;
+                case TracerTarget.Netcoreapp31:
+                    targetFolder = "netcoreapp3.1";
+                    break;
+                case TracerTarget.Net60:
+                    targetFolder = "net6.0";
+                    break;
+            }
+
+            return Path.Combine(_tracerHome, targetFolder, "Datadog.Trace.dll");
```

Comment:
Seeing this code _again_ bothers me - it makes it a pain to add/drop targets! I feel like we should have a single c# file that contains this logic and then we link to it in the project as needs be.

Not for this PR obviously, just a thought 🙂

Reply:
I agree, this can be useful for others.